### PR TITLE
Add notifications for challenge rewards

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description

Added code to display notifications for challenge rewards

### How Has This Been Tested?

Tested in mobile emulator

### CODE REVIEW

File: src/components/notifications/content/ChallengeReward.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+type ChallengeRewardProps = {
+  notification: ChallengeRewardType
+  onGoToRoute: (route: string) => void
+}
+
+const challengeInfoMap: Record<
+  ChallengeRewardID,
+  { title: string; amount: number }
+> = {
+  'profile-completion': {
```
Comment (raymondjacobson) [inline src/components/notifications/content/ChallengeReward.tsx:31]: i wonder if we can/should pull these in via optimizely. e.g. if we change the 7 days to 6 days or something

**Review by raymondjacobson — APPROVED** [review]

File: src/components/notifications/content/ChallengeReward.tsx (hunk 1 above)
Comment (Kyle-Shanks) [reply src/components/notifications/content/ChallengeReward.tsx:31]: 💯 I agree. This same type of config is in 3 places now so I want to put it somewhere central for them all to read from instead

Comment (raymondjacobson) [reply src/components/notifications/content/ChallengeReward.tsx:31]: maybe the client could fetch from discovery

Comment (raymondjacobson) [reply src/components/notifications/content/ChallengeReward.tsx:31]: the more i think about it, the more i like this. wdyt?

Comment (raymondjacobson) [reply src/components/notifications/content/ChallengeReward.tsx:31]: also down to merge this as is first

Comment (Kyle-Shanks) [reply src/components/notifications/content/ChallengeReward.tsx:31]: Cool will merge now and refactor in another PR. And yea I'd like to discuss that approach 👍

**Pull request merged**